webcompat/static/js/lib/models/issue.js

```diff
@@ -231,6 +231,12 @@ issueList.IssueCollection = Backbone.Collection.extend({
       delete params.direction;
     }
 
+    // Issue 999, support domain name search
```

Comment:
nit: We don't typically leave bug numbers for features we're adding, just for bugs that still need to be fixed.